packages/bierzo-wallet/src/logic/transactions/types/BwParserFactory.ts

```diff
@@ -20,6 +20,14 @@ export class BwParserFactory {
     throw new Error('Not supporting generic components yet');
   }
 
+  public static getHeaderRepresentation(tx: any, lastOne: boolean): JSX.Element {
+    if (isSendTransaction({ kind: tx.kind })) {
```

Comment:
This (like in `getReactComponent` and `getCsvRepresentation` as well) makes assumptions on the internal behaviour of `isSendTransaction`. `isSendTransaction` can easily change to check other fields as well.

Better check:

```ts
if (tx.kind === "bcp/send") {
```

in all cases in this file